VIDEO_3_SCRIPT.md

```diff
@@ -0,0 +1,58 @@
+ 
+Hello everyone! Welcome!
+		
+This video will help you understand what AsyncAPI is and how to continue learning about this Open-Source project. 
+
+In this video, we'll talk about:
+
+ - What is AsyncAPI Spec and how did it come about
+ - What AsyncAPI brings
+ - What is AsyncAPI Initiative and what are its objectives
+ - AsyncAPI foundations 
+
+ Ready? Let's get started!
+
+ ## What is AsyncAPI Spec and how did it come about
+
+AsyncAPI is an open-source specification that aims to make working with EDA as easy as it's to work with REST APIs today. But, how to do that?
+
+The Spec is used to describe and document message-driven APIs in a machine-readable format. It’s protocol-agnostic, so you can use it for APIs that work over any protocol (e.g., AMQP, MQTT, WebSockets, Kafka, STOMP, HTTP, etc).
+
+It defines a set of files required to describe an API. These files can then be used to create utilities, such as documentation, integration and/or testing tools.
+
+The AsyncAPI Specification doesn't assume any kind of software topology, architecture or pattern. Therefore, a server MAY be a message broker, a web server or any other kind of computer program capable of sending and/or receiving data. However, AsyncAPI offers a mechanism called "bindings" that aims to help with more specific information about the protocol and/or the topology.
+
+AsyncAPI was founded by Fran Mendez, the Specification started as a side project at Hitch (API Changelog). He identified a lack of tools in the message-driven space and searched for a solution for standardized message-driven specification systems. 
+
+ ## What AsyncAPI brings
+
+Its main contribution to the world of APIs is the standardization of how you can design and describe Event-Driven Architectures in a simplified and unified manner.
+
+We could even say that AsyncAPI is a specification that facilitates, simplifies, and accelerates the work of developers working with APIs. Having defined, documented, and accepted the communication between APIs, AsyncAPI provides an increased and more efficient information control.
+
+AsyncAPI Initiative also develops and maintains an extensive amount of AsyncAPI tools that accelerate application development. The tools' mechanisms for automating complex and essential tasks facilitate code generation, documentation generation, validators, etc. 
+
+## What is AsyncAPI Initiative and what are its objectives
+
+As an open source initiative, AsyncAPI seeks to improve the current state of Event-Driven Architectures (EDA). Its long-term goal is to make working with EDAs as easy as it's to work with REST APIs. That goes from documentation to code generation, from discovery to event management.
+
+AsyncAPI Initiative currently aims to accomplish three main goals:
+
+1. Unify all API specifications. No one does event-driven architecture alone. Most developers complement the event architecture with REST (OpenAPI), GraphQL, and RPC APIs. Let us embrace reusability!
```

Comment:
`Unify all API specifications` it is not about unifying, but reusing. Unify means we would like to have the unified in a kind of way that they look similar, at least this is how I understand this sentence. And also it is not only about API specifications, but others too: `Reuse other specifications` and this `Most developers complement the event architecture with REST (OpenAPI), GraphQL, and RPC APIs. Let us embrace reusability!` you can change to `Most developers complement the event architecture with REST (OpenAPI), GraphQL, or Avro and CloudEvents. Let us embrace reusability!`